src/components/tabs/TabNavigation.tsx

```diff
@@ -37,7 +38,8 @@ const TabNavigation = ({
   selectedIndex,
   tabAlignment = "center",
   onItemPress,
-  children
+  children,
+  excludePadding = false
```

Comment:
How about keeping the same convention used for the `IOScrollViewWith...` component?
```suggestion
  includeContentMargins = true
```